lib/requeue_content.rb

```diff
@@ -20,9 +21,11 @@ def call
 private
 
   def publish_to_queue(edition)
+    version = edition.document.events.last.id
```

Comment:
From a quick glance at this, you shouldn't need to look up the last event of a document, just getting the last event should be sufficient: `Event.maximum(:id)` - and you should look this up before you load the edition as if it's lazy loaded you could be missing an event that just happened.